Add workflow named Quarantine_Endpoint_with_Microsoft_Defender_ATP_from_Slack (#285)
rapid7/insightconnect-workflows: workflows/Quarantine_Endpoint_with_Microsoft_Defender_ATP_from_Slack/help.md

# Description

> This workflow will quarantine an endpoint in Microsoft Defender ATP from a Slack command. This workflow supports an endpoint in the form of an IP address.

Reviewer: ```suggestion
Quarantine an endpoint via IP address in Microsoft Defender ATP from a Slack command.
```
